iOS/Extensions/Dates.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Dates.swift
+//  Aidoku (iOS)
+//
+//  Created by axiel7 on 13/02/2024.
+//
+
+import Foundation
+
+func makeRelativeDate(days: Int) -> String {
```

Comment:
would it make sense to put this as a static function on a `Date` extension or something? just having global functions makes me feel a little weird